code/modules/spacepods/spacepod.dm

```diff
@@ -73,6 +73,8 @@
 	pr_int_temp_processor = new /datum/global_iterator/pod_preserve_temp(list(src))
 	pr_give_air = new /datum/global_iterator/pod_tank_give_air(list(src))
 	equipment_system = new(src)
+	equipment_system.installed_modules = list()
```

Comment:
@TheDZD Moved the definition from the spacepod.dm to equipment.dm now, fixing the defining line and removed the extra line.